lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -27,6 +27,9 @@
       - "vSphere Automation SDK - For tag feature"
       - "vCloud Suite SDK - For tag feature"
     options:
+        datacenter:
+            description: Datacenter name. Required if `search_folder` or `with_folders` are set, otherwise ignored.
```

Comment:
```suggestion
            description:
              - Datacenter name. 
              - Required if C(search_folder) or C(with_folders) are set, otherwise ignored.
```